demo/calculator_blocks/run.py

```diff
@@ -0,0 +1,33 @@
+import gradio as gr
```

Comment:
Adding some very similar demos for the sake of the new examples guide. I think it will help users see how you can add more functionality to the same app little by little but curious on people's thoughts on the matter.

Reply:
Sounds good to me!